hooks/command

```diff
@@ -222,8 +222,7 @@ else
     else
       echo "$pod_json" >&2
 
-      container_status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"
-      if [[ $? -eq 0 ]]; then
+      if ! container_status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"; then
```

Comment:
```suggestion
      if container_status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"; then
```
I think this should be the other way around? that if it succeeded then we want to set the variable